src/components/common/ColonyDecisions/DecisionPreview/DecisionPreview.tsx

```diff
@@ -30,21 +33,10 @@ const MSG = defineMessages({
 
 const DecisionPreview = () => {
   const { user, walletConnecting, userLoading } = useAppContext();
-  const walletAddress = user?.walletAddress || '';
-
-  // Referential equality needed here to avoid infinite loop in effect.
-  const decision = useMemo(
-    () => getDecisionFromLocalStorage(walletAddress),
-    [walletAddress],
+  const decision = useSelector<CoreDecisionsRecord, Decision | undefined>(
```

Comment:
The decision can now be consumed by another part of the app. Note here that sharing component state would be very awkward because it would need to live in the highest ancestor of all the decision components, i.e. our `Routes` file, and be passed down many levels to many components.